### PR TITLE
User app

### PR DESCRIPTION
**Pull request by Irina17191**

### CODE REVIEW

**Comment by Yuruch** [comment]
Maybe add some fixtures?)

**Review by Yuruch — APPROVED** [review]
All good thanks for the work!

**Comment by Irina17191** [comment]
> Maybe add some fixtures?)

yes, sure, I'm doing it already)

**Review by Yuruch — APPROVED** [review]

File: user/views.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from rest_framework.authtoken.views import ObtainAuthToken
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.settings import api_settings
+
+from user.serializers import UserSerializer, AuthTokenSerializer
+
+
+class CreateUserView(generics.CreateAPIView):
+    serializer_class = UserSerializer
+    permission_classes = ()
```
Comment (necobra) [inline user/views.py:11]: maybe (AllowAny,)

**Review by Yuruch — APPROVED** [review]

**Review by BileichukIvan — APPROVED** [review]

**Pull request merged**